eZ/Publish/Core/Repository/PermissionsCriterionHandler.php

```diff
@@ -77,7 +92,7 @@ public function addPermissionsCriterion(Criterion &$criterion)
      */
     public function getPermissionsCriterion($module = 'content', $function = 'read')
     {
-        $permissionSets = $this->repository->hasAccess($module, $function);
+        $permissionSets = $this->PermissionResolver->hasAccess($module, $function);
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/c3a60b9d14944b963ca3998021c5880dfba328bc